src/ui/create.js

```diff
@@ -0,0 +1,54 @@
+var output = require('neat-log/output')
+var stringKey = require('dat-encoding').toStr
+var pretty = require('prettier-bytes')
+var chalk = require('chalk')
+var importUI = require('./components/import-progress')
+
+module.exports = createUI
+
+function createUI (state) {
+  if (!state.dat) {
+    return output`
+    Creating a Dat! Add information to your dat.json file:
+  `
+  }
+
+  var dat = state.dat
+  var stats = dat.stats.get()
+  var title = '\n'
+  var progressView
+  var exitMsg = `
+    Your dat is created! Run ${chalk.green('dat sync')} to share:
+    ${chalk.blue('dat://' + stringKey(dat.key))}
```

Comment:
template literals all the way?

```js
${chalk.blue(`dat://${stringKey(dat.key)}`)}
```

I know it can look confusing to nest them, but it's also weird to mix two concepts that can do the same.